api/hypershift/v1beta1/azure.go

```diff
@@ -284,12 +290,13 @@ type AzureNodePoolOSDisk struct {
 	SizeGiB int32 `json:"sizeGiB,omitempty"`
 
 	// storageAccountType is the disk storage account type to use.
-	// Valid values are Standard, StandardSSD, PremiumSSD and UltraSSD and omitted.
+	// Valid values are Premium_LRS, PremiumV2_LRS, Standard_LRS, StandardSSD_LRS, UltraSSD_LRS.
 	// Note that Standard means a HDD.
 	// The disk performance is tied to the disk type, please refer to the Azure documentation for further details
 	// https://docs.microsoft.com/en-us/azure/virtual-machines/disks-types#disk-type-comparison.
 	// When omitted this means no opinion and the platform is left to choose a reasonable default, which is subject to change over time.
-	// The current default is PremiumSSD.
+	// The current default is UltraSSD_LRS.
```

Comment:
Since this API is multi version, preferably let's don't add a link pinned to 4.18. Also It's not obvious in linked function how that defaults UltraSSD_LRS. In any case the linked function is only used by webhooks which we don't run.

Reply:
@bryan-cox Alberto is right and the default seems to be PremiumSSD although I cant find any documentation to link